checkout-example-advanced/src/main/java/com/adyen/checkout/api/CheckoutResource.java

```diff
@@ -71,16 +70,16 @@ public ResponseEntity<PaymentMethodsResponse> paymentMethods() throws IOExceptio
      * @throws ApiException from Adyen API.
      */
     @PostMapping("/initiatePayment")
-    public ResponseEntity<PaymentsResponse> payments(@RequestHeader String host, @RequestBody PaymentsRequest body, HttpServletRequest request) throws IOException, ApiException {
-        var paymentRequest = new PaymentsRequest();
+    public ResponseEntity<PaymentResponse> payments(@RequestHeader String host, @RequestBody PaymentRequest body, HttpServletRequest request) throws IOException, ApiException {
+        var paymentRequest = new PaymentRequest();
 
         var orderRef = UUID.randomUUID().toString();
         var amount = new Amount()
             .currency("EUR")
             .value(10000L); // value is 10€ in minor units
 
         paymentRequest.setMerchantAccount(this.applicationProperty.getMerchantAccount()); // required
-        paymentRequest.setChannel(PaymentsRequest.ChannelEnum.WEB); // required
+        paymentRequest.setChannel(PaymentRequest.ChannelEnum.WEB); // required
```

Comment:
`channel` is not mandatory so I would remove the comment (`//required`)

Reply:
Good catch! Thanks!